lib/ansible/modules/utilities/logic/async_status.py

```diff
@@ -35,7 +35,6 @@
 notes:
 - This module is also supported for Windows targets.
 seealso:
-- module: async_wrapper
```

Comment:
I'm a bit baffled - the `async_wrapper` module exists, so I don't understand why the `seealso` reference to it would fail.

Reply:
because its not a real module

Reply:
Phantom Module Syndrome . . . that explains it. Thanks @bcoca.